pkg/network/node/ovscontroller.go

```diff
@@ -200,6 +200,7 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	otx.AddFlow("table=90, priority=0, actions=drop")
 
 	// Table 100: egress routing; edited by SetNamespaceEgress*()
+	otx.AddFlow("table=100, priority=300,udp,udp_dst=%d,actions=drop", vxlanPort)
```

Comment:
Just the vxlan port decides that the packet is vxlan?

Reply:
*shrug*. The goal isn't actually "prevent pods from sending VXLAN packets", it's "prevent pods from injecting VXLAN packets into our own VXLAN".